packages/json/package.json

```diff
@@ -45,18 +45,18 @@
     "test": "./build.sh --test && mt dist-for-testing --quiet"
   },
   "dependencies": {
-    "type-fest": "^4.0.0"
+    "type-fest": "^4.2.0"
```

Comment:
Added the types [RequireOneOrNone](https://github.com/sindresorhus/type-fest/blob/v4.2.0/source/require-one-or-none.d.ts) and [UnknownRecord](https://github.com/sindresorhus/type-fest/blob/v4.2.0/source/unknown-record.d.ts).